tests/compute/console.go

```diff
@@ -79,21 +73,24 @@ var _ = SIGDescribe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@re
 				}
 			})
 
-			It("[test_id:1591]should close console connection when new console connection is opened", func() {
+			It("[test_id:1591]should close console connection when new console connection is opened", decorators.Conformance, func() {
 				vmi := libvmops.RunVMIAndExpectLaunch(libvmifact.NewAlpine(), startupTimeout)
+				expectConsoleOutput(vmi, "login")
 
 				By("opening 1st console connection")
-				stream, err := virtClient.VirtualMachineInstance(vmi.Namespace).SerialConsole(vmi.Name, &kvcorev1.SerialConsoleOptions{})
+				stream, err := kubevirt.Client().VirtualMachineInstance(vmi.Namespace).SerialConsole(vmi.Name, &kvcorev1.SerialConsoleOptions{})
 				Expect(err).ToNot(HaveOccurred())
 				defer stream.AsConn().Close()
 
 				firstConsoleErrChan := make(chan error, 1)
 				outReader, outWriter := io.Pipe()
 				inReader, _ := io.Pipe()
 				go func() {
+					defer GinkgoRecover()
```

Comment:
This is only required if the goroutine can fail an assertion, no?

Reply:
Right, I think I messed up with something else

Reply:
Done, PTAL!